app/models/katello/content_view_version_export_history.rb

```diff
@@ -39,11 +40,11 @@ def self.latest(content_view, destination_server: nil)
     end
 
     def export_type_from_metadata
-      cvve.metadata[:incremental] ? INCREMENTAL : COMPLETE
+      metadata[:incremental] ? INCREMENTAL : COMPLETE
     end
 
     def set_export_type
-      self.export_type = export_type_from_metadata
+      self.export_type ||= export_type_from_metadata
```

Comment:
This still seems to me like trying to manage multiple sources of truth.  The export type is stored inside the `metadata` hash, and is also stored in the model's `export_type`.  What if `export_type` were not stored in the db as a separate column, and instead we could just add an `export_type` method to extract the value from `metadata`?  Then you wouldn't have to keep the two values in sync.  Would that cause problems anywhere?

Reply:
This was discussed before but the gist is that we want to be able to query on export_type (`hammer content-export list --type=incremental`.) So it needs to be a column so that we can apply the scoped search magic. That guy ensures that if export_type is not set he pulls from metadata. The actual actions that create this History provide the export_type anyway so this is really not a big deal. These records are not manipulated after in any case.

Reply:
@parthaa Thanks for reminding me of the explanation, makes sense :)